### ISSUE TITLE
Can't create account

### ISSUE THREAD

**Issue by itisNathaniel** [paperwork/paperwork#576]
Hi, 
I set up a new installation of paperwork in a a docker container (from here https://hub.docker.com/r/zuhkov/paperwork/) once I'd set it up with my database I got given the normal login page but whenever I try to create an account I'm given the following alerts
![screen shot 2015-12-19 at 21 12 57](https://cloud.githubusercontent.com/assets/4061191/11915235/562207c6-a695-11e5-968a-e550a0942387.png)

Any ideas as to why this may be?
Thanks!

**Comment by morskipas** [comment]
It looks a little bit strange in your sql . I thinks the userid should not be 0

**Comment by Liongold** [comment]
@itisNathaniel Sorry for the late reply. Did you find a solution for this?

**Comment by itisNathaniel** [comment]
Hi, I didn't no; I'm not sure if the database entries were not created correctly as when a user is created it is not given a 'user_id', it's just left blank. So that's why I'm assuming it's failing to create new users as they can't all be user_id " ".

**Comment by Liongold** [comment]
@itisNathaniel Can you post a screenshot showing the table structure of the users and settings tables, please? (things like field name, length, etc.)

**Comment by mattcurrycom** [comment]
I am also having the same issue, after following the install here:
https://github.com/twostairs/paperwork/wiki/Installation-----CentOS-Fedora

On a VPS.  I have also tried the ubuntu install and it has many more issues. So I figured I would figure out which one you guys "Preferred"; and install that.  The install was much easier on Fedora/Cent, but I was still disappointing it did not work.  If you have a .sql we can run to update this; or just let me know the table. Maybe I can get it to work.
